private/src/scripts/editor/blocks/header/components/DisplayImageData.jsx

```diff
@@ -0,0 +1,27 @@
+class DisplayImageData extends React.Component {
```

Comment:
This should be a functional component.
Since the output is the same as the `DisplayVideoData` component, you could use the same component to output the metadata, and pass in the caption and credit to the component.